R/read_package.R

```diff
@@ -61,7 +63,11 @@ read_package <- function(file = "datapackage.json") {
       )
     }
   }
-  message(msg)
+
+  ## Optionally supress the message
+  if (!isTRUE(quiet)) {
```

Comment:
Why not:

```
if (!quiet) {
``` 

?

Reply:
from `help(isTRUE)`: 

> isTRUE(x) is the same as { is.logical(x) && length(x) == 1 && !is.na(x) && x }; isFALSE() is defined analogously. Consequently, if(isTRUE(cond)) may be preferable to if(cond) because of NAs.

This ensures that the message will be supressed only if the argument is exactly `TRUE` (or converted), not `1`, not `3`,  etc.. `rlang::is_true()` is even more strict I believe.

I wrote it more out of habit than anything else, I don't mind removing `isTRUE()` if you find it more readable that way. If you pass a string to quiet, `isTRUE()` will evaluate to `FALSE`, while `if("any string")` will result in an error: 

```r
Error in if ("any string") { : argument is not interpretable as logical
```

Any preference?